voting/orbs/Elections/processing_vote_test.go

```diff
@@ -77,6 +98,46 @@ func TestOrbsVotingContract_processVote_processVoteMachine_CalulateStakes(t *tes
 	})
 }
 
+func TestOrbsVotingContract_processVote_processVoteMachine_CalulateStakesWithLocked(t *testing.T) {
```

Comment:
I'm not sure what this test tests. every guardian is voting against v2 and indeed it's expelled. but the importance of locked stake is not clear from the code. maybe simplify the scenario or add comments to explain the scenario?

Reply:
compare with a similar func where is the same without lock and now this test also because of lock votes out v5.